### ISSUE TITLE
Azure DevOps Release Pipeline FileTransform Activity Example

### ISSUE THREAD

**Issue by gregl83** [MicrosoftDocs/azure-devops-yaml-schema#181]
Give example of file format in `json` and `xml` files types.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5cca65aa-222a-5789-3821-f5bb31f30013
* Version Independent ID: 5cca65aa-222a-5789-3821-f5bb31f30013
* Content: [FileTransform@1 - File transform v1 task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/file-transform-v1?view=azure-pipelines)
* Content Source: [task-reference/file-transform-v1.md](https://github.com/MicrosoftDocs/azure-devops-yaml-schema/blob/main/task-reference/file-transform-v1.md)
* Service: **azure-devops-pipelines**
* Sub-service: **azure-devops-pipelines-tasks**
* GitHub Login: @steved0x
* Microsoft Alias: **sdanie**

**Comment by WilliamAntonRohm** [comment]
@gregl83 -- Gregory, thank you for your suggestion.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

@steved0x -- Steve, please look into this issue.

**Comment by gregl83** [comment]
Thanks for the links @WilliamAntonRohm!

I did notice that version 2 of the pipeline activity actually includes examples. It's possible this issue was filed hastily but I found version 1 of the docs to be less clear than other documented activities.

For my case I ended up managing variable templating using a python script rather than this activity for two reasons:

- There were already many existing pipelines with tons named variables throughout and I didn't have time to refactor all of them but they weren't using the JSON path expressions (e.g., `FirstElement.SecondElement.ScanDepth`). Renaming could have broken things in this pass.
- Verbosity and explicitness of the variable naming convention "felt" restrictive when applied to a Data Factory ARM templates

Maybe someone else comes to this fork in the road... /shrug
